README.md

```diff
@@ -124,7 +124,7 @@ You can get information on each command with `harpoon help COMMAND`
 * [CIRCL Passive DNS](https://www.circl.lu/services/passive-dns/)
 * [Farsight Dnsdb](https://www.farsightsecurity.com/dnsdb-community-edition/)
 * [FullContact](https://dashboard.fullcontact.com/register)
-* [GreyNoise](https://greynoise.io/)
+* [GreyNoise](https://viz.greynoise.io/account) : supports both Community and Enterprise API.  Use api_type config setting to specify which API type to use.  Both still require an API key to work.
```

Comment:
Some extra details about where to get GN API key and how to define with API to use in the conf.